src/main/java/com/puppycrawl/tools/checkstyle/utils/TokenUtil.java

```diff
@@ -303,6 +303,57 @@ public static boolean isOfType(DetailAST ast, int... types) {
         return ast != null && isOfType(ast.getType(), types);
     }
 
+    /**
+     * Determines if the AST belongs to the given first parameter.
+     *
+     * @param ast the AST node to check
+     * @param first the acceptable token
+     *
+     * @return true if type matches one of the given parameter.
+     */
+    public static boolean isOfType(DetailAST ast, int first) {
```

Comment:
you should add following methods as well:
```
+    public static boolean isOfType(int type, int first) {
+        return type == first;
+    }
+
+    public static boolean isOfType(int type, int first, int second) {
+        return type == first || type == second;
+    }
+
+    public static boolean isOfType(int type, int first, int second, int third) {
+        return type == first || type == second || type == third;
+    }
+
+    public static boolean isOfType(int type, int first, int second, int third, int forth) {
+        return type == first || type == second || type == third || type == forth;
+    }
```

Reply:
Changes Done. Fix Misspelling `"matc"`.